x/docs/internal/keeper/keeper_test.go

```diff
@@ -451,7 +520,107 @@ func TestKeeper_DocumentsIterator_ExistingList(t *testing.T) {
 // --- Document receipts
 // ----------------------------------
 
-func TestKeeper_SaveDocumentReceipt_EmptyList(t *testing.T) {
+func TestKeeper_SaveDocument(t *testing.T) {
+	tests := []struct {
+		name          string
+		empty         bool
+		document      types.Document
+		receipt       types.DocumentReceipt
```

Comment:
What's the difference between receipt and newReceipt?

Reply:
In "existing receipt different Uuid" case (the old function TestKeeper_SaveDocumentReceipt_ExistingReceipt_DifferentUuid) oldReceipt and newReceipt are used